libs/components/forms/src/lib/modules/character-counter/character-counter-indicator.component.html

```diff
@@ -4,12 +4,3 @@
 >
   {{ characterCount + '/' + characterCountLimit }}
 </span>
-<span aria-live="polite" class="sky-screen-reader-only">
```

Comment:
I think with some creative math and an Angular Pipe you could keep this element and keep the code really simple. Make a Pipe that returns the number to announce given the current character count and character limit. The text will only update when the returned value changes and therefore will only be announced on that same interval. You can continue to use the `skyLibResources` pipe to display the message so you don't have to make any changes to the component code.